examples/cisco/8000e/8000e-ixia.pb.txt

```diff
@@ -0,0 +1,130 @@
+name: "cisco-ixia"
+nodes: {
+    name: "vxr"
+    vendor: CISCO
+    model: "8201"
+    os: "ios-xr"
+    config: {
+        image: "ios-xr:latest" # please update the image
+        file: "r1.config"
+    }
+    constraints:{
+        key: "memory"
+        value: "20Gi"
+    }
+    services:{
+        key: 22
+        value: {
+            name: "ssh"
+            inside: 22
+        }
+    }
+    services:{
+        key: 57400
+        value: {
+            name: "gnmi"
+            outside: 57400
+            inside: 57400
+        }
+    }
+   services:{
+        key: 57401
+        value: {
+            name: "gribi"
+            outside: 57401
+            inside: 57400
+        }
+    }
+   services:{
+        key: 57402
+        value: {
+            name: "gnoi"
+            outside: 57402
+            inside: 57400
+        }
+    }
+   services:{
+        key: 57403
+        value: {
+            name: "p4rt"
+            outside: 57403
+            inside: 57400
+        }
+    }
+    interfaces: {
+        key: "eth1"
+        value: {
+          name: "FourHundredGigE0/0/0/0"
+        }
+    }
+    interfaces: {
+        key: "eth2"
+        value: {
+          name: "FourHundredGigE0/0/0/1"
+        }
+    }
+    interfaces: {
+        key: "eth3"
+        value: {
+          name: "FourHundredGigE0/0/0/2"
+        }
+    }
+    interfaces: {
+        key: "eth4"
+        value: {
+          name: "FourHundredGigE0/0/0/3"
+        }
+    }
+}
+
+nodes: {
+    name: "otg"
+    vendor: KEYSIGHT
+    version: "0.0.1-3724" # Please update this with the local version from ixiatg-configmap.yaml
```

Comment:
please use the value "0.0.1-9999" to be consistent with the other examples in this repo

Reply:
done